samples/k8s-offer-azure-vote-custom-meters/mainTemplate.json

```diff
@@ -75,7 +75,7 @@
         },
         "kubernetesVersion": {
             "type": "String",
-            "defaultValue": "1.24.0",
+            "defaultValue": "1.26.0",
```

Comment:
it seems AKS is discontinuing support for 1.26.0 , recently we got issues in our runners (not able to create AKS cluster with K8s' version 1.26.0. I think we should upgrade it to 1.26.3?